imports/include/niFakeNonIvi.h

```diff
@@ -0,0 +1,8 @@
+#ifndef NIFAKE_NON_IVI_H
+#define NIFAKE_NON_IVI_H
+#include <cstdint>
+
+using FakeHandle = uint64_t;
+using int32 = int32_t;
```

Comment:
All drivers have some named handle type, so this needs one too. 

Both grpc and daqmx have a type named `int32`, so we rely on that existing in both. This gives fake non-ivi the same type.